appveyor.yml

```diff
@@ -10,6 +10,12 @@ cache:
   - '%APPVEYOR_BUILD_FOLDER%\swigwin-4.0.1.zip'
   
 environment:
+
+  global:
+    TWINE_USERNAME: geographika
+    TWINE_PASSWORD:
+      secure: 6yuYlaTUIL8SRBakWTke8w==
```

Comment:
not sure how appveyor handles encrypted secret, but I would suspect that the secret should be encrypted with something specific to https://ci.appveyor.com/project/MapServer/mapserver and not your own appveyor account

Reply:
This is an Appveyor encrypted secret of my PyPI password (Twine is the Python library that uploads to PyPI).  I have permissions to upload new packages to https://pypi.org/project/mapscript/. However, I can't add new maintainers (e.g. a new "mapserver" user) though as I'm not an admin. Maybe @tomkralidis does?